app/controllers/surveys_controller.rb

```diff
@@ -68,7 +68,7 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_survey
-      @survey = Survey.find_by_uuid(params[:uuid])
+      @survey = Survey.find_by(uuid: params[:uuid])
```

Comment:
Why

Reply:
Uhm I have no idea why this got changed back @spencerldixon unless I needed to merge with master before I pushing this? I will revert this change immediately.